apps/condo/domains/property/components/panels/Builder/MapJsonSchema.json

```diff
@@ -115,6 +123,120 @@
+                                                "title": "label",
+                                                "type": "string"
+                                            },
+                                            "name": {
+                                                "title": "name",
+                                                "type": "string"
+                                            },
+                                            "type": {
+                                                "enum": [
+                                                    "unit"
+                                                ],
+                                                "title": "type",
+                                                "type": "string"
+                                            },
+                                            "unitType": {
+                                                "enum": [
+                                                    "flat",
+                                                    "parking"
+                                                ],
+                                                "title": "unitType",
+                                                "type": "string"
+                                            }
+                                        },
+                                        "required": [
+                                            "id",
+                                            "label",
+                                            "type"
+                                        ],
+                                        "type": "object"
+                                    },
+                                    "title": "units",
+                                    "type": "array"
+                                }
+                            },
+                            "required": [
+                                "id",
+                                "index",
+                                "name",
+                                "type",
+                                "units"
+                            ],
+                            "type": "object"
+                        },
+                        "title": "floors",
+                        "type": "array"
+                    },
+                    "id": {
+                        "title": "id",
+                        "type": "string"
+                    },
+                    "index": {
+                        "title": "index",
+                        "type": "number"
+                    },
+                    "name": {
+                        "title": "name",
+                        "type": "string"
+                    },
+                    "type": {
+                        "enum": [
+                            "section"
+                        ],
+                        "title": "type",
+                        "type": "string"
+                    }
```

Comment:
Maybe to uplift it to top to be consistent with other nodes of declaration "tree"?